.github/workflows/all-green.yml

```diff
@@ -0,0 +1,30 @@
+name: Check Pull Request CI Status
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+permissions:
+  checks: read
+  statuses: read
+
+jobs:
+  all-jobs-are-green:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Ensure CI Success
+        uses: DataDog/ensure-ci-success@v1
```

Comment:
We've built something similar to replace https://github.com/upsidr/merge-gatekeeper in our CI - https://github.com/lendable/sloth - maybe you would be interested in joining the effort? 😊

Reply:
gosh, I crawl the web to find an action that fits my needs, and I missed yours 🤕 .

Unfortunately, I won't be able to join, as go is chineese to me.

Though, I'll put some suggestion easy to tackle that would have been a blocker for me. If at  some point it's a good fit, I'll consider using yours!

Thank's a lot for the head's up!

Reply:
No worries, ours is the latter one though (sloth), which is in TypeScript :)